### ISSUE TITLE
Crash when using offline mode

### ISSUE THREAD

**Issue by amaurydavid** [at-internet/atinternet-apple-sdk#44]
Hello :)
For some time now my app uses ATInternet v2.8.9 with offline mode.
I've recently decided to update the SDK with the latest version (2.11.2) but now the app crashes at start because of the offline mode :(

Here is the way I setup ATInternet:
```
let tracker = ATInternet.sharedInstance.defaultTracker

    tracker.setConfig(
      [ TrackerConfigurationKeys.Secure: "true",
        TrackerConfigurationKeys.Site: ATInternetTagging.siteId,
        TrackerConfigurationKeys.Log: ATInternetTagging.logServer,
        TrackerConfigurationKeys.LogSSL: ATInternetTagging.logSercureServer,
        TrackerConfigurationKeys.HashUserId: "false",
        TrackerConfigurationKeys.Identifier: IdentifierTypeKey.idfv.rawValue,
        TrackerConfigurationKeys.OfflineMode: OfflineModeKey.required.rawValue],
    override: false)
```

The app crashes in Storage.swift l115 with a `Fatal error: Unexpectedly found nil while unwrapping an Optional value`

```
let managedObjectModel: NSManagedObjectModel? = {
        let bundle = Bundle(for: Tracker.self)
        let modelPath = bundle.path(forResource: "Tracker", ofType: "momd")
        let modelURL = URL(fileURLWithPath: modelPath!) // <- here is the bad boy
        
        return NSManagedObjectModel(contentsOf: modelURL)
    }()
```

I've tried to delete the app and reinstall it, still crashes.
Note that it doesn't crash if I don't use the offline mode option in my tracker configuration.

More env infos: Xcode 10.1.0, iOS simulator 12.1.

**Comment by nsagnett** [comment]
Hello ! 

I don't understand because this code was removed since 2.9.0 sdk version

Maybe an issue on dependency process ?

**Comment by amaurydavid** [comment]
Indeed.
For some reason Cocoapods was thinking it was using the 2.11.2, but the info plist was telling me it was still the 2.8.9. Now it's fixed, the 2.11.2 is used and it no longer crashes :)
